src/rpc/trans/http/RpcOnHttpSocket.cpp

```diff
@@ -49,6 +51,22 @@ void RpcOnHttpSocket::send(const DataPackage& package) throw(IOException)
 		request->setRequestUrl(PATH_RPC_SERVICE);
 		header = request;
 	}
+#else
+	//����ͷ��
+	ScopePointer<HttpHeader> header = null;
```

Comment:
use ScopePointer instead of SmartPtr due to there is a segment fault bug with SmartPtr.